test/jdk/java/net/httpclient/HttpResponseInputStreamInterruptTest.java

```diff
@@ -104,12 +107,12 @@ public void test() {
 
     static class Handler implements HttpHandler {
 
-        CountDownLatch countDownLatch;
-        CountDownLatch interruptLatch;
+        CountDownLatch interruptReadyLatch;
+        CountDownLatch interruptDoneLatch;
```

Comment:
should be `final`

Reply:
I believe this change was missed and is pending?